src/components/MatomoOptOut.tsx

```diff
@@ -53,31 +31,43 @@ const MatomoOptOut: React.FC<IProps> = () => {
     localStorage.setItem(MATOMO_LS_KEY, String(!checked))
   }
   return (
-    <Container>
-      <p>
+    <Flex
+      border="1px solid"
+      borderColor="border"
+      bgColor="background"
+      borderRadius={1}
+      p={6}
+      direction="column"
+      mb={4}
+      mt={8}
+      alignItems="flex-start"
+      justifyContent="space-between"
+    >
+      <Text color="fail">
         You can opt out of being tracked by Matomo Analytics and prevent the
         website from analysing the actions you take using the website. This will
         prevent us from learning from your actions and creating a better website
         experience for you and other users.
-      </p>
+      </Text>
       {loading ? (
         "Loading preferences..."
       ) : (
-        <form>
-          <input
+        <FormControl m={0}>
```

Comment:
not sure if this is working as expected. In https://ethereumorgwebsitedev01-immiguelpethereumorgwebsi61534.gtsb.io/en/cookie-policy/ I don't see the input
![image](https://user-images.githubusercontent.com/468158/192854033-7447cae7-07b0-41fd-9cfa-74c3a7364e7a.png)